### ISSUE TITLE
Hugo updates

### ISSUE THREAD

**Issue by curiousdannii** [angstsmurf/spatterlight#33]
I just noticed you've done a lot of work on the Hugo interpreter. Do you think the updates would work in other Glk interpreters?

**Comment by angstsmurf** [comment]
Not really. The Hugo code in [hecugel.c](https://github.com/angstsmurf/spatterlight/blob/master/terps/hugo/heglk/hecugel.c) bypasses the Glk screen model with parent windows that split into child windows, and instead uses custom windows with arbitrary size and position. And it still needs lots of game-specific hacks to make games like Cryptozookeeper and Future Boy work with screen resizing and variable window margins.

The text colour code uses your zcolors API, however, so that might be possible to transplant to Gargoyle. I'm not super motivated to do that myself due to the messy state of the code, but perhaps somebody else might be able to find something useful in there.

**Comment by curiousdannii** [comment]
Gotcha. Eventually I want to work on getting more graphics support into Lectrote's Hugo interpreter, so I'll look then at what you've done here.
